### ISSUE TITLE
YITH Gift Cards Compatibility

### ISSUE THREAD

**Issue by angelleye** [angelleye/paypal-woocommerce#949]
* https://mail.google.com/mail/u/0/#inbox/16008e87d1ab851f

When applying a gift card on an order with this YITH Gift Cards plugin we end up with a 10427 error from PayPal about order totals not adding up.  

User will be providing a copy of the plugin for us to work with.  Waiting on that...

**Comment by angelleye** [comment]
[yith-woocommerce-gift-cards-premium (1).zip](https://github.com/angelleye/paypal-woocommerce/files/1515197/yith-woocommerce-gift-cards-premium.1.zip)

**Comment by angelleye** [comment]
@kcppdevelopers I gave this copy of our dev branch but he says it's not fixing his problem..??

**Comment by kcppdevelopers** [comment]
@angelleye Could you please ask API request log file for more troubleshoot ?

**Comment by iMansoorAliKhan** [comment]
@kcppdevelopers @angelleye , Here is ticket relavent to this thread along with the login detail 
https://www.angelleye.com/support/scp/tickets.php?id=1662

**Comment by kcppdevelopers** [comment]
@angelleye @iMansoorAliKhan I have double checked client website, they are not using our dev branch. they should used dev branch to resolved calculation issue.

**Comment by iMansoorAliKhan** [comment]
Another report for this issue : https://www.angelleye.com/support/scp/tickets.php?id=1679,
Sent them the dev branch, will leave feedback  here

**Comment by iMansoorAliKhan** [comment]
**Update**
The client applied the dev branch and error was fixed but now they are seeing another error. Below is the detailed scenario of the error.

> Thank you! I’m not getting a 10427 error anymore. 
> 
> However I am getting a 10426 error now - ‘item total invalid’

> It’s at checkout when attempting to complete the order. The same point at which I was recording the 10427 error before.
> 
> Example: I have an order for $25.67 ($15 item, $10.67 shipping) I used a gift card for $25 leaving $.67 to run on the credit card. After entering payment details and hitting submit, the error displays at the top of the page.
> 

@kcppdevelopers  can you tell what could be the reason here?

**Comment by angelleye** [comment]
@iMansoorAliKhan Need to see the log so we can see the full request.

**Comment by iMansoorAliKhan** [comment]
@angelleye , I have shared that with you, and its related to shipping amount in negative. I have created another issue for that.

**Comment by angelleye** [comment]
@kcppdevelopers It looks like your commit from above was included in our 1.4.7 release, so now I'm a little confused if this was actually resolved or not..??

**Comment by iMansoorAliKhan** [comment]
@angelleye , this issue was resolved with the dev branch we shared with the client, but then they reported another issue which I have referenced here #973  and also shared logs with @kcppdevelopers .

**Comment by angelleye** [comment]
Okay, then I'm going to leave this issue closed.
